discovery-provider/src/models.py

```diff
@@ -479,6 +480,70 @@ def __repr__(self):
 created_at={self.created_at},\
 updated_at={self.updated_at}"
 
+class DailyUniqueUsersMetrics(Base):
+    __tablename__ = "daily_unique_users_metrics"
+
+    id = Column(Integer, primary_key=True)
+    count = Column(Integer, nullable=False)
+    timestamp = Column(Date, nullable=False)
```

Comment:
timestamp here should be zero'd out to the day right? -- maybe add a comment

Reply:
maybe call the attribute 'day'? and 'month' for the monthly tables?